src/java/fr/paris/lutece/portal/business/file/TemporaryFileHome.java

```diff
@@ -140,4 +140,22 @@ public static TemporaryFile findByPrimaryKey( int nKey )
     {
         return _dao.findByUser( user );
     }
+    
+    /**
+     * Delete the temporary files older than the given number of days
+     * @param days
+     */
+    public static void deleteFilesOlderThan( int days )
+    {
+        List<TemporaryFile> files = _dao.selectFilesOlderThan( days );
```

Comment:
Does this load the file data ? this seems wasteful.

Reply:
No only the physical file id